javascript/embedded-login/.env.example

```diff
@@ -1,7 +1,7 @@
-AM_URL=$AM_URL
+SERVER_URL=$SERVER_URL
```

Comment:
Do the mentions of "AM_URL" also need updating in the `webpack.config.js` file?
e.g. `const AM_URL = process.env.AM_URL;`